packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset/reset.html

```diff
@@ -44,5 +44,27 @@
             >
             </oui-select>
         </oui-field>
+        <!-- private network -->
+        <oui-field
+            data-label="{{ :: 'kubernetes_add_private_network_label' | translate }}"
+        >
+            <oui-select
```

Comment:
This looks like it could be a reusable component as logic looks similar and duplicated

Reply:
True, i think it's bit too late to change this.
The prod is schedule for tomorrow.
Any changes will force us to review and QC again.

But i will create a Jira to fix this later.
Thanks.

Reply:
Jira created.